File: app/src/main/java/com/yhjoo/dochef/classes/Notice.java
package com.yhjoo.dochef.classes;

public class Notice {
    public final String title;
    public final String contents;
    public final long dateTime;

    public Notice(String title, String contents, long dateTime) {
        this.title = title;
        this.contents = contents;
        this.dateTime = dateTime;
    }

    public String getTitle() {
        return title;
    }

    public String getContents() {
        return contents;
    }

    public long getDateTime() {
        return dateTime;
    }
}

File: app/src/main/java/com/yhjoo/dochef/data/model/Notice.kt
package com.yhjoo.dochef.data.model

import com.google.gson.annotations.SerializedName

class Notice(
    @SerializedName("title") val title: String,
    @SerializedName("contents") val contents: String,
    @SerializedName("datetime") val dateTime: Long
)